src/Validator/DocumentValidator.php

```diff
@@ -273,4 +298,19 @@ public static function isValidLiteralValue(Type $type, $valueNode)
 
         return $context->getErrors();
     }
+
+    public static function assertValidSDLExtension(DocumentNode $documentAST, Schema $schema)
+    {
+        $errors = self::visitUsingRules($schema, new TypeInfo($schema), $documentAST, self::sdlRules());
+        if (count($errors) !== 0) {
+            throw new Error(
+                implode(
+                    '\n\n',
```

Comment:
not sure whats better, personally I'd use PHP_EOL constants twice (`PHP_EOL.PHP_EOL`)

Reply:
Actually, we don't use carriage returns anywhere, so `\n` is appropriate. PHP_EOL is not for our case. I'll revert it.